pytest_xprocess.py

```diff
@@ -15,14 +15,14 @@ def pytest_addoption(parser):
 
 
 def getrootdir(config):
-    return config.rootdir.join(".xprocess").ensure(dir=1)
+    return config.cache.makedir(".xprocess")
 
 
 def pytest_cmdline_main(config):
     xkill = config.option.xkill
     xshow = config.option.xshow
     if xkill or xshow:
-        # config._do_configure()
```

Comment:
Can we remove this?

Reply:
This will make [test_functional_workflow](https://github.com/pytest-dev/pytest-xprocess/blob/master/example/test_server.py#L75) fail with

> Traceback (most recent call last):
>  ...
>   File ".../pytest_xprocess.py", line 18, in getrootdir
>   return config.cache.makedir(".xprocess")
> AttributeError: 'Config' object has no attribute 'cache' 

do you think it would be better to remove it and then alter the test?

Reply:
Sorry I meant the commented out line. 😁

Reply:
Hm, I don't think I get it. In our current master branch this line is commented, but my PR uncomments it so that [test_functional_workflow](https://github.com/pytest-dev/pytest-xprocess/blob/master/example/test_server.py#L75) passes. If the line is removed it will fail with the traceback I have showed above. 

I'm not 100% this is the case but, apparently, `cacheprovider` plugin won't load for this particular test unless `config._do_configure` is called (safe to say that at the very least Config will not hold a reference to the cache object).

For reference, here is the method implementation from [pytest docs](https://docs.pytest.org/en/stable/_modules/_pytest/config.html#Config)

```python
def _do_configure(self) -> None:
    assert not self._configured
    self._configured = True
    with warnings.catch_warnings():
        warnings.simplefilter("default")
        self.hook.pytest_configure.call_historic(kwargs=dict(config=self))
```

On a side note, we have some repeated code in our test suite and we are still under 100% coverage. I'm considering a PR to address it (unless there are other things we should focus on that I'm not seeing). WDYT?

Reply:
Argh sorry, of course that's fine, I completely misread the PR. Time to go to bed I guess. 😁

Reply:
Sorry about the confusion!

Reply:
Don't worry about it.

> Time to go to bed I guess. 😁
That makes two of us...

Thanks for the prompt feedback!